contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -4673,3 +4680,23 @@ is_function_pg_stat_valid(FunctionCallInfo fcinfo, PgStat_FunctionCallUsage *fcu
 
 	pgstat_end_function_usage(fcu, finalize);
 }
+
+static SortByNulls
+unique_constraint_nulls_ordering(ConstrType constraint_type, SortByDir ordering)
+{
+	if (sql_dialect == SQL_DIALECT_TSQL && constraint_type == CONSTR_UNIQUE)
+	{
+		switch (ordering)
+		{
+			case SORTBY_ASC:
+			case SORTBY_DEFAULT:
+				return SORTBY_NULLS_FIRST;
+			case SORTBY_DESC:
+				return SORTBY_NULLS_LAST;
+			default:
+				return SORTBY_NULLS_DEFAULT;
```

Comment:
what other ordering will have in here ?

Reply:
`SORTBY_USING` is the only other type, but that option is never possible here.
I am actually reusing this bit of code from here [link](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/3fcd5ba305c108539a368f67c6a789b72e71dc57/contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-epilogue.y.c#L1893),